src/loaders/resourceLoader.ts

```diff
@@ -318,12 +318,8 @@ export abstract class ResourceLoader implements IResourceBase {
     }
 
     if (shouldClearSubject) {
-      // Clear out the cache for the whole of the schema registry, which
-      // right now is just the subjects.
-      const schemaRegistry = await this.getSchemaRegistryForEnvironmentId(schema.environmentId);
```

Comment:
Here's the call that is no longer needed now that a Subject can be used to identify what needs to be cache cleared.